packages/lib-sourcify/src/lib/SourcifyChain.ts

```diff
@@ -239,4 +323,45 @@ export default class SourcifyChain {
         this.chainId
     );
   };
+
+  getContractCreationBytecode = async (
+    address: string,
+    transactionHash: string
+  ): Promise<string> => {
+    const txRecipt = await this.getTxRecipt(transactionHash);
+    const tx = await this.getTx(transactionHash);
```

Comment:
What's the difference between the tx and the reciept?

Reply:
from the ethers docs: 

* A **TransactionReceipt** includes additional information about a transaction that is only available after it has been includfed in the block.
* A **TransactionResponse** includes all properties about a transaction that was sent to the network, which may or may not be included in a block.